ddmc.json

```diff
@@ -0,0 +1,140 @@
+            "schema": {
+                "type": "string",
+                "subtype": "band-name"
+            },
+            "default": "cirrus",
+            "optional": true
+        },
+	{
+            "name": "swir16",
+            "description": "The name of the SWIR (ca 1600 nm) band. Defaults to the band that has the common name `swir16` assigned.\n\nEither the unique band name (metadata field `name` in bands) or one of the common band names (metadata field `common_name` in bands) can be specified. If the unique band name and the common name conflict, the unique band name has a higher priority.",
+            "schema": {
+                "type": "string",
+                "subtype": "band-name"
+            },
+            "default": "swir16",
+            "optional": true
+        },
+	{
+            "name": "swir22",
+            "description": "The name of the SWIR (ca 2200 nm) band. Defaults to the band that has the common name `swir22` assigned.\n\nEither the unique band name (metadata field `name` in bands) or one of the common band names (metadata field `common_name` in bands) can be specified. If the unique band name and the common name conflict, the unique band name has a higher priority.",
+            "schema": {
+                "type": "string",
+                "subtype": "band-name"
+            },
+            "default": "swir22",
+            "optional": true
+        },
+        {
+            "name": "target_band",
+            "description": "By default, the dimension is the band dimension. You can specify a new dimension name in this parameter so that a new dimension label with the specified name will be added for the computed values.",
+            "schema": [
+                {
+                    "type": "string",
+                    "pattern": "^\\w+$"
+                },
+                {
+                    "type": "null"
+                }
+            ],
+            "default": "band",
+            "optional": true
+        },
+	{
+            "name": "gain",
+            "description": "The value by which the indices are to be multiplied. By default, gain is 2.5.",
+            "schema": [
+                {
+                    "type": "double",
+                    "pattern": "^\\w+$"
+                },
+                {
+                    "type": "null"
+                }
+            ],
+            "default": 2.5,
+            "optional": true
+        }
+    ],
+    "returns": {
+        "description": "A raster data cube containing the computed DDMC values. The structure of the data cube differs depending on the value passed to `target_band`:\n `target_band` is a string: The data cube keeps the same dimensions. The dimension properties remain unchanged, but the number of dimension labels for the dimension of type `bands` increases by one. The additional label is named as specified in `target_band`.",
```

Comment:
The dimension properties remain unchanged, but the number of dimension labels for the dimension of type `bands` increases by one. I think the number of dimension labels becomes 3, because the labels you create are the "MIDCL", "LCL" and "DC".